src/main/resources/templates/browse.html

```diff
@@ -2,95 +2,264 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Browse</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Browse Movies - Movie Recommender</title>
     <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" rel="stylesheet">
     <style>
-        .list
-        {
-            display: flex;
-            flex-direction: column;
-            align-items: center;
-            padding: 0;
-            list-style-type: none;
-        }
-        .list-item
-        {
-            text-align: center;
-            margin: 10px 0;
+        :root {
+            --primary-color: #e74c3c;
+            --secondary-color: #2c3e50;
+            --background-light: #f8f9fa;
+        }
+
+        body {
+            background-color: var(--background-light);
+        }
+
+        .navbar {
+            background-color: rgba(255, 255, 255, 0.95) !important;
+            box-shadow: 0 2px 4px rgba(0,0,0,.1);
+        }
+
+        .navbar-brand {
+            font-weight: bold;
+            color: var(--secondary-color) !important;
+        }
+
+        .nav-link {
+            color: #34495e !important;
+            margin: 0 10px;
+            transition: color 0.3s ease;
+        }
+
+        .nav-link:hover {
+            color: var(--primary-color) !important;
```

Comment:
once again just changed the UI to make it more pleasing to the user. I Used bootstrap as well